src/applications/check-in/components/PreCheckinConfirmation.jsx

```diff
@@ -31,45 +37,33 @@ const PreCheckinConfirmation = props => {
           {t('youve-completed-pre-check-in')}
         </h1>
         <AppointmentBlock appointments={appointments} />
-        {hasUpdates ? (
-          <va-alert
-            background-only
-            status="info"
-            show-icon
-            data-testid="confirmation-update-alert"
-          >
-            <div>
-              {t(
-                'a-staff-member-will-help-you-on-the-day-of-your-appointment-to-update-your-information',
-              )}
-            </div>
-          </va-alert>
-        ) : (
-          <></>
-        )}
-        <p className={hasUpdates ? `vads-u-padding-left--2` : ``}>
+        <p className="vads-u-margin-bottom--4">
           <ExternalLink
             href="https://va.gov/health-care/schedule-view-va-appointments/appointments/"
             hrefLang="en"
           >
-            {t('go-to-your-appointment')}
```

Comment:
This is another deprecated key it looks like?

Reply:
Yep, I removed it. One thing to note is that this one made it in this round of translation, but the new key `go-to-your-appointment-details` didn't. It might be good to sub in the translation of `go-to-your-appointment` for Spanish in the mean time until we get the next round of translations.

Reply:
Oh good call. I'll make a note of that for my translation ticket.